datahub-web-react/src/app/lineage/types.ts

```diff
@@ -81,7 +81,7 @@ export type NodeData = {
     canEditLineage?: boolean;
     upstreamRelationships?: Array<LineageRelationship>;
     downstreamRelationships?: Array<LineageRelationship>;
-    health?: Health[];
+    health?: Maybe<Health[]>;
```

Comment:
Why was this change necessary? Can we replace usage with `health={health ?? undefined}` or something like that instead?

Reply:
Nvm, seems fine